src/opm/parser/eclipse/EclipseState/Schedule/Well/Well.cpp

```diff
@@ -698,12 +698,22 @@ bool Well::canOpen() const {
         if (prod.WaterRate.is<std::string>())
           return true;
 
-        return ((prod.OilRate.get<double>() + prod.GasRate.get<double>() + prod.WaterRate.get<double>()) != 0);
+        if (!prod.OilRate.zero())
+            return true;
+
+        if (!prod.GasRate.zero())
+            return true;
+
+        if (!prod.WaterRate.zero())
+            return true;
+
+        return true;
```

Comment:
Should this (line 710) be `return false` instead?  As of right now, `canOpen()` appears to unconditionally return `true` for producers.

Reply:
Yes thank you - it should return `false `